.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Describe what this PR does and why we need it**:
```

Comment:
Could we also include an issue template? WIth steps to reproduce, environment and so on.